openff/interchange/_tests/unit_tests/interop/lammps/export/test_lammps.py

```diff
@@ -62,3 +68,104 @@ def test_to_lammps_single_mols(self, mol, sage_unconstrained, n_mols):
+            mol.add_conformer(
+                (conf_centered + 2 * r_eff * int_offset).to("nm")
+            )  # space copied by effective diameter
+            tiled_top.add_molecule(mol)
+
+        ## 3a) set periodic box tightly around extremem positions in filled topology
+        box_dims = tiled_top.get_positions().ptp(axis=0)
+        box_vectors = (
+            numpy.eye(3) * box_dims
+        )  # convert to diagonal matrix to get proper shape
+
+        # EXPORT TO LAMMPS
+        interchange = Interchange.from_smirnoff(
+            sage_unconstrained, tiled_top
+        )  # , charge_from_molecules=[pilot_mol])
+        # UNRELATED QUESTION: why are waters not correctly recognized?
+        # PDB residue name on output is always UNK for Molecule.from_smiles("O"), EVEN when using tip3p.offxml)
+        interchange.box = box_vectors
+        interchange.to_lammps(temp_lammps_path)
+
+        # EXTRACT ATOM INFO FROM WRITTEN LAMMPS FILE TO TEST IF MOLEUCLE IDS ARE BEING WRITTEN CORRECTLY
+        with temp_lammps_path.open(
+            "r"
+        ) as lmp_file:  # pull out text from temporary lammps file ...
+            all_lines = [
+                line for line in lmp_file.read().split("\n") if line
+            ]  # ... separate by newlines, remove empty lines ...
+            atom_lines = all_lines[
+                all_lines.index("Atoms") + 1 : all_lines.index("Bonds")
+            ]  # ... extract atoms block ...
+        temp_lammps_path.unlink()  # ...and finally, unceremoniously kill the file once we're done with it
+
+        ## SIDENOTE: would be nice if there was an easier, string-parsing-free way to extract this info
+        ## Could enable some kind of Interchange.from_lammps() functionality in the future, perhaps
+        def extract_info_from_lammps_atom_lines(atom_line: str) -> dict[str, Any]:
+            """Parse atom info from a single atom-block linds in a .lmp/.lammps files"""
+            KEYWORDS: dict[str, type] = {  # reference for the distinct
+                "atom_index": int,
+                "molecule_index": int,
+                "atom_type": int,
+                "charge": float,
+                "x-pos": float,
+                "y-pos": float,
+                "z-pos": float,
+            }
+
+            return {
+                field_label: FieldType(str_val)
+                for str_val, (field_label, FieldType) in zip(
+                    atom_line.split("\t"), KEYWORDS.items()
+                )
+            }
+
+        written_mol_ids: set[int] = set()
+        for atom_line in atom_lines:
+            atom_info = extract_info_from_lammps_atom_lines(atom_line)
+            written_mol_ids.add(atom_info["molecule_index"])
+        expected_mol_ids = {
+            i + 1 for i in range(interchange.topology.n_molecules)
+        }  # we'd like for each of the N molecules to have ids from [1...N]
+
+        return expected_mol_ids == written_mol_ids
```

Comment:
```suggestion
        assert expected_mol_ids == written_mol_ids
```

I think my useless ` -> None:` annotation in a different test suggested that this should return a boolean of the pass/fail state, but `pytest`-style tests are meant to not return anything and just run without errors (read: fail on various `assert` statements) synonymously with passing